src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,99 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+//import mate.jdbc.service.DeleteAll;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        //DeleteAll.delAll();
```

Comment:
i neew this for delete all data from all tables

Reply:
I don't suppose, that this comments are needed here. You can have them in your local IDE, but the should not be in the PR